shoulder_pitch_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: shoulder_pitch_joint
    joint_speed: 0.2
    motor:
        id: 1
        init: 512
        min: 0
        max: 1023

elbow_pitch_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: elbow_pitch_joint
    joint_speed: 0.2
    motor:
        id: 2
        init: 512
        min: 0
        max: 1023

wrist_pitch_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: wrist_pitch_joint
    joint_speed: 0.2
    motor:
        id: 3
        init: 512
        min: 0
        max: 1023

gripper_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: gripper_joint
    joint_speed: 0.2
    motor:
        id: 4
        init: 512
        min: 0
        max: 1023

